helm/values.yaml

```diff
@@ -0,0 +1,6 @@
+num_trackers: 3
```

Comment:
helm values variables should be camel case

Reply:
you might find it cleaner to split out the values config into trackers, build_index, origins, etc. 

i.e:
```
trackers:
  replicas: 3

build_index:
  replicas: 3

origins:
  replicas: 3
```

see: https://github.com/helm/charts/blob/master/incubator/istio/values.yaml

Reply:
👍